### ISSUE TITLE
Load only filtered type

### ISSUE THREAD

**Issue by Copylefter** [Vestride/Shuffle#85]
Is it possible to load by ajax new items only that type that was selected before by filter?

For example i have homepage were will be several types of posts:books, news, reviews. More posts can be loaded by ajax. Also i have a filter option that will show only books or news or reviews. 
Is it possible to load by ajax only books if it selected in filter?

**Comment by Vestride** [comment]
Using the `append` method, new items should be filtered automatically to match your last filter. Is that not happening?

**Comment by Copylefter** [comment]
Thanx, i didn't try yet.
